sky/shell/manifest.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_MANIFEST_H_
+#define SKY_SHELL_MANIFEST_H_
+
+#include <string>
+
+#include "base/memory/scoped_ptr.h"
+#include "base/version.h"
+#include "url/gurl.h"
+
+namespace sky {
+namespace shell {
+
+struct Manifest {
```

Comment:
I would probably create a `//sky/shell/updater` directory and put the code related to updating in there.  It doesn't quite seem right for this class to be at the top level in shell.  That would also give you a place to put the platform independent code for the update task.